knowledge-base/window-backdrop-blur.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Apply a backdrop filter with a blur effect to the Modal Window background
+description: How to apply a backdrop filter with a blur effect to the background content of the Modal Window.
+type: how-to
+page_title: Apply a backdrop filter with a blur effect to the Modal Window background
+slug: window-kb-backdrop-blur
+position: 
+tags: 
+ticketid: 1642061
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Window for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+* How do I blur the background in Windows?
+* Is it possible to apply a backdrop filter with a blur effect on the background content of the Window?
+
+## Solution
+
+Override the built-in CSS styles of the background with the code below to adjust the blur amount. 
+
+>caption This code will affect all modal Dialogs and Windows on the web page.
```

Comment:
Treat captions like titles, they must extract the essence and be concise. This is a complete sentence.